src/frontend/common/header/Category.js

```diff
@@ -0,0 +1,27 @@
+import './header.css';
+import { Link } from 'react-router-dom';
+import { PRODUCTS } from '../../../routes';
+
+const categoryList = [
+  'Protein',
+  'Gym Gear',
+  'Athletics',
+  'Nutrition',
+  'Vegan',
+  'Clothing'
+];
```

Comment:
Don't add data here above the component. Add it in a separate file and then import.

Reply:
this will be removed in final PR

Reply:
cool